<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tally Desktop - Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBoxMediaPlayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>360</width>
     <height>215</height>
    </rect>
   </property>
   <property name="title">
    <string>Media Player</string>
   </property>
   <widget class="QRadioButton" name="radioCLMP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Command line</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioPhonon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Qt</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabMediaPlayerOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPhonon">
     <attribute name="title">
      <string>Qt</string>
     </attribute>
     <widget class="QPushButton" name="phononChangeScreen">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>80</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Screen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>5</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Use this button to control the screen on which the Qt media player appears.  If the button is inactive, click 'Apply' to activate it.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCLMP">
     <attribute name="title">
      <string>Command line</string>
     </attribute>
     <widget class="QLineEdit" name="argsCLMP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathCLMP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelPathCLMP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path to media player</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelArgsCLMP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Command line arguments (executed after input media list)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxWebAddress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>360</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>Default Web Address</string>
   </property>
   <widget class="QLineEdit" name="defaultWebAddress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="settingsOK">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsApply">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
